eqcorrscan/core/lag_calc.py

```diff
@@ -292,8 +297,18 @@ def xcorr_pick_family(family, stream, shift_len=0.2, min_cc=0.4,
                 if (tr.stats.station, tr.stats.channel) \
                         not in pick_stachans:
                     template_plot.remove(tr)
+            if plotdir is not None:
+                plotdir = "{plotdir}/{template_origin}/".format(plotdir=plotdir
```

Comment:
So this should probably be:
```python
plotdir = "{plotdir}/{template_origin}/".format(
    plotdir=plotdir, template_origin=event.resource_id.id[0:19])
```
Is there a reason for taking just the 0-19th parts of the template event resource id?  I think that `event` should probably also be replaced by `family.template.event` - this will give you the template events origin id.

Reply:
I would like to make a sorted of sub directory of templates' origin that can access easy and simple, to find how many detected a template.
But i find now that it is unnecessary. Because plot save and sort with name of event's id and templates' origin in a folder.
So i remove this part and all image will save in a folder. Like your suggestion.
thank you for your patience and guide :)